cybersixgill_actionable_alerts/README.md

```diff
@@ -0,0 +1,67 @@
+# Agent Check: cybersixgill_actionable_alerts
+
+## Overview
+By integrating Cybersixgill actionable alerts, Datadog customers gain a premium,
+automated threat intelligence solution based on the most comprehensive data sources from the deep, dark and surface web. 
+It is customizable, enabling users to define key assets relevant to their brand, industry, and geolocation. Users can covertly 
+monitor critical assets such as IP addresses, domains, vulnerabilities, and VIPs for activity on the underground and closed sources - and 
+prioritize, as well as respond to threats directly from the Siemplify dashboard.
```

Comment:
Can you provide more insight on the type of threats? 
```suggestion
Cybersixgill actionable alerts allows you to monitor critical assets like IP addresses, domains, vulnerabilities, and VIPs within Datadog to respond to threats from the dark web. 
```

Reply:
Examples of threat types are Data Leak, Malware, Phishing, Fraud, Vulnerability Exploit, Insider Threat, Web Attack